ngrinder-controller/src/main/java/org/ngrinder/agent/service/AgentPackageService.java

```diff
@@ -167,23 +169,14 @@ public File createMonitorPackage() {
 				addFolderToTar(tarOutputStream, libPath);
 				final URLClassLoader classLoader = (URLClassLoader) getClass().getClassLoader();
 				Set<String> libs = getMonitorDependentLibs(classLoader);
+				copyShellFile(tarOutputStream , basePath ,"monitor" );
```

Comment:
please follow intellij coding style.

Reply:
Yes, I will apply code formatting to whole files before merging this PR.